main.js

```diff
@@ -0,0 +1,50 @@
+const tweeter = Tweeter();
+const renderer = Renderer();
+
+renderer.renderPosts(tweeter.getPosts());
+
+function post() {
+  if ($("#input").val()) {
+    tweeter.addPost($("#input").val());
```

Comment:
create a variable the will contain the input value, instead of accessing the dom twice